shared/fs/filepreview/image-view.native.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+import * as React from 'react'
+import {globalStyles} from '../../styles'
+import {NativeImage} from '../../common-adapters/native-wrappers.native'
+import {type ImageViewProps} from './image-view'
+
+const ImageView = (props: ImageViewProps) => (
+  <NativeImage source={{uri: props.url}} style={stylesImage} resizeMode="contain" />
```

Comment:
i wonder if its safer to use the webview instead, we don't have to change it but i wonder if its more isolated that way

Reply:
I used this because I was under the impression that this was safer. What are risks of images that we are thinking of?

Reply:
the webview is more isolated than a regular component inside our hierarchy i would assume